tests/helpers/rapidsim.py

```diff
@@ -53,7 +52,7 @@ def analyze_histo(histo):
 
 
 def load_generated_histos(file_name, particles):
-    with uproot4.open(file_name) as rapidsim_file:
+    with uproot.open(file_name) as rapidsim_file:
```

Comment:
Are you sure about this? The problem is that if uproot moves to e.g. uproot5, this may breaks (as happened with uproot3). So I would suggest to keep it pinned. It's better to get deprecation warnings than having a new uproot release that will implicitly break it.

Reply:
That would be better yes. 033640f chooses uproot4 instead of uproot.